lib/u3d/installation.rb

```diff
@@ -90,8 +123,26 @@ def path
     end
 
     def packages
-      PlaybackEngineUtils.list_module_configs(root_path).map do |mpath|
-        PlaybackEngineUtils.module_name(mpath)
+      pack = []
+      PlaybackEngineUtils.list_module_configs(root_path).each do |mpath|
+        pack << PlaybackEngineUtils.module_name(mpath)
+      end
+      %w[Documentation StandardAssets MonoDevelop].each do |module_name|
```

Comment:
Let's move that to a module or Installation class constant. Maybe something like SPECIAL_PACKAGES? Better name idea?